frontend/src/scenes/experiments/ExperimentView/Info.tsx

```diff
@@ -49,7 +52,13 @@ export function Info(): JSX.Element {
         return <></>
     }
 
-    const lastRefresh = legacyPrimaryMetricsResults?.[0]?.last_refresh
+    // Get the last refresh timestamp from either legacy or new results format
+    // Check both primary and secondary metrics for the most recent timestamp
+    const lastRefresh =
+        legacyPrimaryMetricsResults?.[0]?.last_refresh ||
+        legacySecondaryMetricsResults?.[0]?.last_refresh ||
+        primaryMetricsResults?.[0]?.last_refresh ||
+        secondaryMetricsResults?.[0]?.last_refresh
```

Comment:
I'm ok with this, but it's not checking the most recent. Shouldn't it be the oldest timestamp instead?